core/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ add_library(
   "${CMAKE_CURRENT_SOURCE_DIR}/src/process_stash.cpp"
   "${CMAKE_CURRENT_SOURCE_DIR}/src/session.cpp"
   "${CMAKE_CURRENT_SOURCE_DIR}/src/transport.cpp"
+  "${CMAKE_CURRENT_SOURCE_DIR}/src/openid.cpp"
```

Comment:
Move openid.cpp file between multipart_form_data.cpp and process_stash.cpp

Reply:
Done.